phpcs.xml.dist

```diff
@@ -7,6 +7,14 @@
 
     <rule ref="Cdn77"/>
 
+    <rule ref="SlevomatCodingStandard.TypeHints.UnionTypeHintFormat">
+        <properties>
+            <property name="withSpaces" value="no" />
+            <property name="shortNullable" value="no" />
```

Comment:
Personally I don't like this one, I don't see a reason for it ...

Reply:
@simPod suggested it might be deprecated so, it would be good to forward with it..

Reply:
`?` says `or null` which is actually a union so there's no real point for having an exception for `null` type

_having `?` deprecated is only my with_ 🙃

Reply:
Well, it's the most used union type, so having a shortcut for it is nothing wrong I think...It's shorter, everybody immediately knows what it means, ... I like it :) I vote to force shortNullable where applicable.

Reply:
But if more people are against it, I'll be ok with this rule. I don't like it but I can live with it.

Reply:
few more points

- `?` makes it harder to extend unions. 
```diff
- ?int
+ int|string|null
```
vs
```diff
- int|null
+ int|string|null
```

- `?` was introduced because there could not be proper unions in php at the time so now it seems redundant

Reply:
I'm OK with shortNullable = no

Reply:
I'm OK with shortNullable = no